demo/portal/src/app/config.js

```diff
@@ -14,11 +13,7 @@ var angular = require('angular');
  * @constructor
  */
 function AppConfig($stateProvider, $urlRouterProvider) {
-  var redirectUrlForInvalidRoutes = '/unauthorised';
-
-  $urlRouterProvider.otherwise(function() {
-    return redirectUrlForInvalidRoutes;
-  });
+  $urlRouterProvider.otherwise('/workorders/list');
```

Comment:
don't like this change, this will cause a unauthorised user to start firing requests which will 403 for him.

![image](https://user-images.githubusercontent.com/3279642/30800248-50adaf18-a1d7-11e7-9d68-eb0e7fed5326.png)

Reply:
That's fine. This change is just simplification for the documentation we have.
We going to document this that whatever ui you will get you can configure it to land on any other page and change otherwise to route you want. I fully get your point but keep in mind that we going to bring and merge your other PR after release. 

For the moment we want to minimize amount of changes that may affect documentation and also have some side effects. We can also remove stack traces from 403 errors.

Reply:
> For the moment we want to minimize amount of changes that may affect documentation 
that's fine

> and also have some side effects. We can also remove stack traces from 403 errors.
I'd rather leave them there as a reminder to do it correctly at some stage

Reply:
I just do not want to overcomplicate things when we will need to get role, resource based auth straight after to support things properly. We will also move portal to angular4 which will differ a lot.